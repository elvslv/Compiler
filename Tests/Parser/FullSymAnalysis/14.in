var i, j: integer;
var arr: array[1..10, 1..11] of integer;
const m = 12.35;
begin
	if arr[i] then
		i := j;
end.