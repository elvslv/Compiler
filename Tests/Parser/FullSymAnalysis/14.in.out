var I : INTEGER;
var J : INTEGER;
var ARR : array [const 1..const 10] of array [const 1..const 11] of INTEGER;
const M = 12.35;

11 5 Condition must be integer